db/migrate/20160624142936_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration
+  def change
+    create_table :comments do |t|
+      t.text :content, null: false
```

Comment:
I've run into problems in the past with setting null constraints on text columns.  IIRC, it's mostly MySQL who falls on its face so you might not ever run into it.  Just wanted to point that out.